src/context/theme.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {
+  createContext,
+  FunctionComponent,
+  useCallback,
+  useContext,
+  useState,
+} from "react";
+import {
+  CustomTheme,
+  govWalletTheme,
+  sallyTheme,
+} from "../common/styles/themes";
+
+interface ThemeContext {
+  theme: CustomTheme;
+  setTheme: (themeName: string | undefined) => void;
+}
+
+export const ThemeContext = createContext<ThemeContext>({
+  theme: sallyTheme,
+  setTheme: () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+});
+
+export const ThemeContextProvider: FunctionComponent = ({ children }) => {
+  const [theme, setThemeValue] = useState(sallyTheme);
+
+  const setTheme = useCallback(
+    (themeName?: string) => {
+      if (themeName === "GOVWALLET") {
+        setThemeValue(govWalletTheme);
+      } else {
+        setThemeValue(sallyTheme);
+      }
+    },
+    [setThemeValue]
+  );
+
+  return (
+    <ThemeContext.Provider value={{ theme: theme, setTheme }}>
```

Comment:
Just noticed, can we do the following? Or does eslint complain?
```suggestion
    <ThemeContext.Provider value={{ theme, setTheme }}>
```

Reply:
oh yes we can! changed [here](https://github.com/rationally-app/mobile-application/pull/493/commits/ff9c4c50fd7bd9304fc5a2e589e259806460ef2b)!